docs/cli/cli-spans.md

```diff
@@ -0,0 +1,38 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+operations to attributes on a Span.
+
+The **span name** SHOULD be of the format `{process.executable.name}`. In case this is
+not informative enough because of interpreters calling other files, e.g. `python script.py` or in
```

Comment:
how would generic instrumentation know whether an arbitrary string is informative or not?

Reply:
The history of this was not generic, rather in context of click which would have a better value I think. How might we clarify this better? https://github.com/open-telemetry/semantic-conventions/issues/1577

Reply:
Yep, the context of this is building an instrumentation for a library (click) used for building CLIs. So there are cases where we have more clue :)